test/app.test.js

```diff
@@ -24,132 +24,269 @@ describe("app", function() {
-            it("should ask them to put or post", function() {
+            it("should ask them to pick a route", function() {
                 return tester
                     .start()
                     .check.interaction({
                         state: 'states:start',
                         reply: [
-                            'Choose your destiny:',
-                            '1. Put something',
-                            '2. Post something',
-                            '3. Cause an error'
+                            'Welcome to CTA train tracker.Pick a route:',
+                            '1. Red Line',
+                            '2. Blue Line',
+                            '3. Brown Line',
+                            '4. Green Line',
+                            '5. Orange Line',
+                            '6. Purple Line',
+                            '7. Pink Line',
+                            // '8. Yellow Line',
+                            '8. Exit'
                         ].join('\n')
                     })
                     .run();
             });
+        });
 
-            describe("when the user chooses to put", function() {
-                it("should ask them to put something", function() {
-                    return tester
-                        .setup.user.state('states:start')
-                        .input('1')
-                        .check.interaction({
-                            state: 'states:put',
-                            reply: 'What would you like to put?'
-                        })
-                        .run();
-                });
+        describe("when the user is asked to pick a route e.g red line", function() {
+            it("should select red line", function() {
+                return tester
+                    .setup.user.state('states:red')
+                    .input('1')
+                    .check(function(api) {
+                        var req = api.http.requests[0];
+                        assert.deepEqual(req.params, {rt: 'red',
+                        							  key: '33305d8dcece4aa58c651c740f88d1e2',
+                        							  outputType: 'JSON'});
+                    })
+                    .run();
             });
 
-            describe("when the user chooses to post", function() {
-                it("should ask them to post something", function() {
-                    return tester
-                        .setup.user.state('states:start')
-                        .input('2')
-                        .check.interaction({
-                            state: 'states:post',
-                            reply: 'What would you like to post?'
-                        })
-                        .run();
-                });
+            it("should tell them the result", function() {
+                return tester
+                    .setup.user.state('states:red')
+                    .input('1')
```

Comment:
These tests should probably set the user up in the beginning of the interaction, not in `states:red`, since that is a delegation state and doesn't take any input. It should put the user in `states:start`, and then the `1` input will be the user's selection for red.